code/modules/mob/living/carbon/xenomorph/abilities/boiler/boiler_macros.dm

```diff
@@ -5,6 +5,13 @@
 	var/action_name = "Bombard"
 	handle_xeno_macro(src, action_name)
 
+/datum/action/xeno_action/verb/verb_toggle_gas_type()
```

Comment:
for the both of these, spacing between the `set`s and the action itself would be nice